airflow/www/templates/airflow/graph.html

```diff
@@ -268,5 +274,17 @@
         return false
     }
 
+    d3.select("#refresh_button").on("click",
+        function() {
+            $.get(
```

Comment:
We need some sort of indicator that things are getting fetched here, maybe the loading.gif count else where